pulsar-broker/src/main/java/com/yahoo/pulsar/broker/admin/BrokerStats.java

```diff
@@ -169,7 +169,7 @@ public LoadReport getLoadReport() throws Exception {
             @PathParam("cluster") String cluster, @PathParam("namespace") String namespace) throws Exception {
         try {
             NamespaceName ns = new NamespaceName(property, cluster, namespace);
-            SimpleLoadManagerImpl lm = (SimpleLoadManagerImpl) (pulsar().getLoadManager());
+            SimpleLoadManagerImpl lm = (SimpleLoadManagerImpl) (pulsar().getLoadManager().get());
```

Comment:
Since this method is used only for testing I changed it to only execute if `lm instanceof SimpleLoadManager Impl` and return an empty map otherwise (should not happen).